schema/v1.1.0/dataset_config_models_extended.py

```diff
@@ -227,9 +227,9 @@ def valid_dates(self) -> Self:
 # ID Object Validation
 # ==============================================================================
 @alru_cache
-async def validate_id(id: str) -> Tuple[dict, bool]:
+async def validate_id(id: str) -> Tuple[List[str], bool]:
```

Comment:
now returns synonyms and whether or not it is valid, not returning extra unnecessary data. this also makes it more modular, with the ability to swap this out for wormbase validation against a different api endpoint